src/index.js

```diff
@@ -270,7 +271,10 @@ function bookRoom(event) {
   const booking = {
     userID: currentUser.id,
     date: searchDate,
-    roomNumber: roomNumber
+    roomNumber: parseInt(roomNumber),
+    roomServiceCharges: []
   }
-  console.log(booking)
+  hotelData.bookings.bookings.push(booking)
```

Comment:
Way to remember to update your bookings! I've been struggling with how to do this in my own project... I had only considered running my fetch method again to update hotelData...